test/fixtures/security-context-deprecation.md

```diff
@@ -0,0 +1,178 @@
+# Logic for deprecating securityContext.enabled
```

Comment:
I think it'd be best to include this file under the 'design' directory - it'll be a helpful reference to link people to, and less buried! 😄

Feel free to copy it over as-is (without changing the content).

You should include the current date in the filename, e.g. `20191217-filename.md`.

Reply:
File copied as requested 😄

Reply:
Can you also remove the file from this directory? I should have said 'move it over' 😅

Reply:
No worries. Done.